### DISCUSSION TITLE
MSDRG

### DISCUSSION THREAD

**Discussion by ACLubin** [CMSgov/hospital-price-transparency#52] [Q&A] [answered]
If a hospital has MS-DRG as payment methodology for some payers, however, does not have established MS-DRG charges with associated local code, how would that be reflected on the file?
![image](https://github.com/CMSgov/hospital-price-transparency/assets/158351988/51903e18-ee00-40ff-b0b9-9fb6745df9b2)

**Comment by RidenourK** [comment, marked as answer]
@ACLubin. Thank you for your question. 

As indicated in the notes concerning code types in the “documentation” section of this CMS Hospital Price Transparency - Data Dictionary GitHub repository, hospital items and services may be associated with one or more billing codes or accounting codes. You should include in your MRF any code(s) used by the hospital for purposes of billing or accounting for the item or service. Examples include Current Procedural Terminology (CPT), Healthcare Common Procedure Coding System (HCPCS), National Drug Code (NDC), Revenue Center (RC) code, or other common payer identifier. The list of valid values is in the table with the name of the standard and the associated valid values. The value "LOCAL" may be used for internal accounting codes in conjunction with another billing code for that item or service. However, if no other code types are available for a particular item or service, "LOCAL" may be used as a valid value.

**Reply by ACLubin** [reply]
In that scenario, is it fine to leave the standard_charge|gross and standard_charge|discounted_cash cells blank?

**Reply by RidenourK** [reply]
@ACLubin. Thank you for your question. 

As indicated at [45 CFR 180.50(b)(2)](https://www.ecfr.gov/current/title-45/part-180#p-180.50(b)(2)(iii)(C)), each hospital must encode in its machine-readable file all standard charge information, **as applicable**, for the required data elements. 

In the “General CSV Instructions” of this GitHub repository, we provide the following guidance: Do not insert a value or any type of indicators (e.g., “N/A”) if the hospital does not have applicable data to encode. If you would like to include an explanation for the blanks, you may do so using Additional Generic Notes or Additional Payer-Specific Notes.
